test/11_lib_conv2_image.t

```diff
@@ -0,0 +1,52 @@
+#! /usr/bin/perl
+
+use strict;
+use File::Temp;
+use Test::More;
+
+use vars '$Script';
+$Script = "test";
+
+use MakeExt2Image;
+use Conv2Image;
+
+die("you must be root to run this test") if $>;
```

Comment:
https://metacpan.org/pod/Test::More suggests
```suggestion
plan skip_all => "you must be root to run this test" if $>;
```

Reply:
Yes but that doesn't make the tests fail. I wanted it to fail to make a point.